client/src/store/index.ts

```diff
@@ -21,5 +22,6 @@ export default new Vuex.Store<RootStateI>({
     checkInOnExpire: false
   },
   actions: rootActions,
-  mutations: rootMutations
+  mutations: rootMutations,
+  getters: Getters
```

Comment:
If you imported these things "as" their property name, this would be even cleaner. Like this:
https://github.com/bcgov/business-filings-ui/blob/2fbb3c2973c7d6454305034312efa5b1960c9c7e/src/store/index.ts#L14

Reply:
I don't disagree! 
I'll look into it 👍

Reply:
Updated in next commit!